nixos/doc/manual/release-notes/rl-2105.xml

```diff
@@ -91,6 +96,12 @@
   </para>
 
   <itemizedlist>
+   <listitem>
+    <para>
+     The X11 Plasma session name is now <literal>plasma</literal>, not <literal>plasma5</literal>.
```

Comment:
Please add in code the rename so that people with aliases enabled don't need to do anything.